_translator-files/po/de/Privacy-Statement.po

```diff
@@ -5,14 +5,16 @@
 #
```

Comment:
As noted in the other PR, I'd be in favour of not translating this.

Reply:
Should I copy the english PO to DE?

And keep this as "de"

`#. type: YAML Front Matter: lang
#: ../wiki/en/Installation-for-iOS.md:1
#, no-wrap
msgid "en"
msgstr "de"
`

Reply:
Maybe a german version has to be provided:

https://www.datenschutz.org/datenschutzerklaerung-mehrsprachig/

Reply:
> Should I copy the english PO to DE?
> 
> 
> 
> And keep this as "de"
> 
> 
> 
> `#. type: YAML Front Matter: lang
> 
> #: ../wiki/en/Privacy-Statement.md:1
> 
> #, no-wrap
> 
> msgid "en"
> 
> msgstr "de"
> 
> `

No. If we decide to not translate the file, leave it untranslated (this will correctly mark the content as en not de for accessibility reasons. 

Concerning the translation, if it's required, we should leave it. Just maybe we'd need to add that the English version is what one should refer to.

Reply:
Please decide :)

Reply:
Ok. I think we should add a disclaimer that this is just a translation, and that the english one is the binding one.

Reply:
Added "(Die englische Version ist massgeblich.)"

Reply:
Ok. Thanks. Opened #751